src/main/features/response/routes/claim-details.ts

```diff
@@ -1,15 +1,24 @@
 import * as express from 'express'
 import { Paths } from 'response/paths'
+import { Paths as ClaimPaths } from 'claim/paths'
 import { Claim } from 'claims/models/claim'
 import { getInterestDetails } from 'shared/interestUtils'
 import { ErrorHandling } from 'shared/errorHandling'
+import { User } from 'idam/user'
+
+function isCurrentUserLinkedToClaim (user: User, claim: Claim): boolean {
+  return claim.defendantId === user.id
+}
 
 /* tslint:disable:no-default-export */
 export default express.Router()
-  .get(Paths.claimDetailsPage.uri, ErrorHandling.apply(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
-    const claim: Claim = res.locals.claim
-    const interestData = await getInterestDetails(claim)
-    res.render(Paths.claimDetailsPage.associatedView, {
-      interestData: interestData
+  .get(Paths.claimDetailsPage.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+      const claim: Claim = res.locals.claim
+      const interestData = await getInterestDetails(claim)
+      res.render(Paths.claimDetailsPage.associatedView, {
+        interestData: interestData,
+        pdfUrl: isCurrentUserLinkedToClaim(res.locals.user, res.locals.claim) ? ClaimPaths.sealedClaimPdfReceiver : ClaimPaths.receiptReceiver
```

Comment:
why is there an if statement here? the only person who should see this page should be the defendant so the else statement should never be hit? and if it is not the defendant that is viewing this page shouldn't one of the page guards have redirected them on page load?

Reply:
@robertparkinson that page is accessible from defendant task list and both dashboards.

In the future we plan to move it to claim feature as that page has nothing to do with response. Access will remain the same.